index.php

```diff
@@ -36,6 +36,7 @@
 }
 
 $wantsurl = empty($SESSION->wantsurl) ? '' : $SESSION->wantsurl;
+$noredirect  = optional_param('noredirect', 0, PARAM_BOOL); // Don't redirect.
```

Comment:
Should the default value be `false` here instead?

It's functionally the same thing, though might be more semantically correct.